test/e2e/accounts/common.ts

```diff
@@ -264,17 +270,23 @@ export async function signData(
 ) {
   const isAsyncFlow = flowType !== 'sync';
 
-  await switchToOrOpenDapp(driver);
+  // This step can frequently fail, so retry it
+  await retry(
+    {
+      retries: 3,
+      delay: 2000,
+    },
+    async () => {
+      await switchToOrOpenDapp(driver);
 
-  await driver.clickElement(locatorID);
+      await driver.clickElement(locatorID);
 
-  // behavior of chrome and firefox is different,
-  // chrome needs extra time to load the popup
-  if (driver.browser === 'chrome') {
-    await driver.delay(500);
-  }
+      // take extra time to load the popup
+      await driver.delay(500);
```

Comment:
Was behavior for firefox inconsistent here?

Reply:
Yes, Firefox was now the major offender

Reply:
Hmm I wonder why that is, it would be cool to see how we've changed/added delays over the course of browser releases. Irrelevant to this immediate PR though.